src/components/Loading/LoadingIcon.jsx

```diff
@@ -2,8 +2,16 @@ import React, { PureComponent } from "react";
 import PropTypes from "prop-types";
 import classNames from "classnames";
 import { isNumber, noop } from "lodash";
+import { generateId } from "utils";
 
 class LoadingIcon extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: generateId(4),
+    };
```

Comment:
no need to use the state to store the id.
 Replace the `constructor` block with following code.
```
id = generateId(4)
```